fastestimator/op/numpyop/numpyop.py

```diff
@@ -223,8 +223,8 @@ def set_rua_level(self, magnitude_coef: float) -> None:
     def forward(self, data: List[np.ndarray], state: Dict[str, Any]) -> Union[np.ndarray, List[np.ndarray]]:
         return self.fn(*data)
 
-    def forward_batch(self, data: Union[Tensor, List[Tensor]], state: Dict[str,
-                                                                           Any]) -> Union[np.ndarray, List[np.ndarray]]:
+    def forward_batch(self, data: Union[torch.Tensor, List[torch.Tensor]],
+                      state: Dict[str, Any]) -> Union[np.ndarray, List[np.ndarray]]:
```

Comment:
We should keep this as `data: Union[Tensor, List[Tensor]]` because it could be np array